CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change log
+
+## Version 2.1.1
```

Comment:
These are typically formatted as a link with the date, as well as pointing to a compare for all the code changes:

https://raw.githubusercontent.com/auth0/auth0-PHP/master/CHANGELOG.md

Reply:
@joshcanhelp This is great - thanks!